rpmlb/custom.py

```diff
@@ -19,20 +18,16 @@ def run_cmds(self, key, **kwargs):
         if 'name' in kwargs:
             env['PKG'] = kwargs['name']
 
-        for cmd in self.each_yaml_cmd(key):
-            utils.run_cmd(cmd, env=env)
-
-    def each_yaml_cmd(self, key):
-        cmd_dict = self._get_yaml_content(self._file_path)
+        cmd_dict = self._get_yaml_content()
         if key not in cmd_dict:
             return
-        cmds = cmd_dict[key]
-        for cmd in cmds:
-            yield cmd
+        cmd_element = cmd_dict[key]
+
+        Yaml.run_cmd_element(cmd_element, env=env)
 
-    def _get_yaml_content(self, file_path):
+    def _get_yaml_content(self):
```

Comment:
This is asking for a `@property`:

```python
@property
def yaml_content(self):
    # The body stays the same
```

Usage: `cmd_dict = self.yaml_content`

Generally, any `get_anything(self)` method (getter with only `self` parameter) should be a `@property`.